_data/banking.yml

```diff
@@ -428,18 +456,21 @@ websites:
     - name: PenFed
       url: https://www.penfed.org/
       twitter: PenFed
+      facebook: PenFed
       img: penfed.png
       tfa: No
 
     - name: PNC Bank
       url: https://www.pnc.com/
       twitter: pncnews
+      facebook: pncbank
       img: pncbank.png
       tfa: No
 
     - name: Postbank
```

Comment:
Perhaps change the `url` to `https://www.postbank.de/` and add `lang: de` for Postbank?